_schema.graphql

```diff
@@ -1157,6 +1158,14 @@ type ArtworkContextFair {
     # A tz database time zone, otherwise falls back to `X-TIMEZONE` header
     timezone: String
   ): String
+  active_start_at(
+    # This arg is deprecated, use timezone instead
+    convert_to_utc: Boolean
```

Comment:
<!--
  1 failure:  Addition uses `sn...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 1163;
-->
- :no_entry_sign: Addition uses `snake_case` instead of `camelCase`: `convert_to_utc`